### ISSUE TITLE
search input in multiple -tra

### ISSUE THREAD

**Issue by darekzak** [arendjr/selectivity#207]
Is it possible to have search input in dropdown for multiple inputType? Because from what I see when showSearchInputInDropdown is set to true search input shows, but after selecting any item, dropdown closes, but there no items displayed as selected. When I reopen dropdown, i can see that previously selected item is missing in dropdown list, so i guess that there is some problem with displaying it in container for selected items. Or maybe for some reason search isn't allowed in multiple, but if so then what's the reason for it?

**Comment by darekzak** [comment]
Ok, I think i can manage to position search input for multiple in dropdown by styles/custom templates. But still i think that this situation with showSearchInputInDropdown: true in multiple is a bug.

**Comment by arendjr** [comment]
Search inputs in dropdowns are not supported for multiple input types because the searching is supposed to be done in the input in the selector itself. To be honest I'm a bit confused of what you're trying to achieve... do you have a screenshot of what you intend to do?

**Comment by darekzak** [comment]
I'll try to describe the whole process that got me to the bug/strange behaviour

Goal: search input to be displayed in dropdown
1. showSearchInputInDropdown: true as an option to multiple contructor
2. dropdown opens - search input is displayed in dropdown, the same as in single select
3. select any item from dropdown
4. dropdown closes 
5. If before opening dropdown there were some items displayed as selected then those items now are gone. There is empty select.

Then I read in documentation that this option showSearchInputInDropdown: true is meant to be used only in single input type. So if it has no meaning for multiple input, then i guess that passing it as one of the options shouldn't  lead to the situation that i described. I would rather expect not showing search input in dropdown, or throwing some error/warning in console if in options there is something that has no impact on plugin.

If you'll consider it as bug, I would be happy to try to fix it.

**Comment by arendjr** [comment]
I agree not showing it, possibly with a warning (in dev builds only) would be more desirable here. PR welcome :)
